scripts/libs.js

```diff
@@ -354,7 +358,16 @@ export function mergeUniqueArrays(...arrays) {
 }
 
 export function getBaseUrl() {
```

Comment:
We have removed the usage of basepath in this PR https://github.com/hlxsites/henkel-raqn-guide/pull/29

Should this be brought back?

Reply:
Yes, since on the other SharePoint configs are different, making things break when we have fixed paths like before